src/transports/auth_negotiate.c

```diff
@@ -14,8 +14,12 @@
 #include "auth.h"
 #include "git2/sys/cred.h"
 
+#ifdef __APPLE__
+#import <GSS/GSS.h>
```

Comment:
Is the API really completely the same? That's pleasant

Reply:
It's… strange. The `/usr` headers have their symbols marked as deprecated in favor of GSS, so this makes sure we don't select the `/usr` ones. And there's a more CoreFoundation-style API available as well, which is different. But using the old symbols while linking to GSS is fine 🤷‍♂.